src/raftstore/store/peer.rs

```diff
@@ -1723,14 +1784,9 @@ impl Peer {
     }
 
     pub fn get_min_progress(&self) -> u64 {
-        let status = self.raft_group.status();
-        status
-            .progress
-            .values()
-            .chain(status.learner_progress.values())
-            .map(|pr| pr.matched)
-            .min()
-            .unwrap_or_default()
+        self.raft_group.status_ref().progress.map_or(0, |p| {
+            p.iter().map(|(_, pr)| pr.matched).min().unwrap_or_default()
```

Comment:
not consider learner progress?

Reply:
status_ref.progress contains both voters and learners.

Reply:
it's really confusing, why not make them consistent in raft-rs?

Reply:
To not break the API in a patch version and also make TiKV 3.0 depends on a stable version of raft.